src/app/auth/auth.js

```diff
@@ -0,0 +1,152 @@
+angular.module('prx.auth', ['prx.ui.nav'])
+.config(function (PRXDrawerProvider) {
+  PRXDrawerProvider.register({
+    name: 'Sign In',
+    weight: PRXDrawerProvider.BOTTOM,
+    href: 'http://www.prx.org/sessions/new',
+    nav: true,
```

Comment:
nav is another relative value for the priority with which it is kept out of the drawer. In this case, because the value is `true`, it will be the last thing to be removed from the nav and put in the drawer.